cves/CVE-2013-2115.yml

```diff
@@ -162,37 +184,43 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
-    note:
+    applies: true
+    note: |
+      if the OGNL successfully got through, the privileges of the user
+      were changed. It shows the importance of preventing this XSS problem, and
+      the importance of sanitizing input.
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      This was majorly due to XSS. The original fix did not sanitize or even
```

Comment:
Again, why was it majorly due to XSS? Additionally, 'majorly' is informal and the sentence may sound better if refactored.

Reply:
See previous answer to comment